js/PulsarFormComponent.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 
 var $ = require('jquery'),
-    TimePickerComponent = require('./TimePickerComponent');
+TimePickerComponent = require('./TimePickerComponent');
```

Comment:
~tabbing~ Actually it looks like this branch also contains the JQ3 changes, might be better to merge that branch first so the JQ3 changes are separated out as this PR is big enough already!